x-pack/plugins/stack_alerts/public/alert_types/threshold/index.ts

```diff
@@ -18,9 +18,7 @@ export function getAlertType(): AlertTypeModel<IndexThresholdAlertParams> {
       defaultMessage: 'Alert when an aggregated query meets the threshold.',
     }),
     iconClass: 'alert',
-    documentationUrl(docLinks) {
-      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/alert-types.html#alert-type-index-threshold`;
-    },
+    documentationUrl: (docLinks) => docLinks.links.alerting.indexThreshold,
```

Comment:
Now that the documentation for the index threshold and es query stack alert types are on their own pages, can we update the doc links to link directly to those pages instead of to the alert type pages?

Reply:
Changed the link properly.